wiki/608039-DateAnnotationLeaderboard.md

```diff
@@ -2,10 +2,14 @@
 {row}
 {column width=12}
 
+- Latest schema version: `1.1.2`
```

Comment:
This was originally added below the leaderboard widget, but the physical address has two leaderboards, so I added it to the top.  Decision is to create a dedicated section below.